server/src/main/java/io/pravega/schemaregistry/service/SchemaRegistryService.java

```diff
@@ -657,7 +656,7 @@ private boolean isValidCompatibilityForFormat(SerializationFormat serializationF
                                                                                GroupProperties groupProperties) {
         switch (groupProperties.getCompatibility().getType()) {
             case AllowAny:
-                return CompletableFuture.completedFuture(Collections.emptyList());
+                return getSchemasForAllowAnyPolicy(namespace, group, schema, groupProperties);
```

Comment:
you dont need a separate method here.. you can just use ```store.listLatestSchemas(namespace, group);```

this will fetch the latest schemas and the validation actually checks for the "allowmultipletypes" check.